package.json

```diff
@@ -29,7 +29,8 @@
     "uswds-gulp": "github:uswds/uswds-gulp#5e32f1b99279a6e50ad875e66cc4cb77b456c390",
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0",
-    "watchify": "^3.11.1"
+    "watchify": "^3.11.1",
+    "y18n": "^5.0.5"
```

Comment:
This shouldn't be added to the package.json unless it's a direct dependency of our app. We don't want to add any constraints to the version, we just want npm to resolve an acceptable version based on the other dependencies pulling it in. You should be able to just delete this line.

Reply:
The dependencies are what is causing it to pull an old version and make us vulnerable: https://app.snyk.io/org/data.gov/project/0c896621-a417-4b2b-95e1-14d0954b222a. I was testing to see if the various dependencies are compatible with the latest version. I already know we can't upgrade uswds, one of the libraries pulling in y18n... Looks like another ticket then to update lots of dependencies. https://github.com/GSA/resources.data.gov/issues/156